Mission 3/NMCA_003/NMCA_003_strings.lua

```diff
@@ -1,9 +1,455 @@
+}
+
+-- Reminder #1 to kill Aeon base
+M1AeonBaseReminder1 = {
+    {text = '[Benson]: The Aeon base is still a threat, destroy it.', vid = 'N03_M1AeonBaseReminder1.sfd', bank = 'N03_VO', cue = 'M1AeonBaseReminder1', faction = 'UEF'},
+}
+
+-- Reminder #2 to kill Aeon base
+M1AeonBaseReminder2 = {
+    {text = '[Benson]: The base wont kill itself commander.', vid = 'N03_M1AeonBaseReminder2.sfd', bank = 'N03_VO', cue = 'M1AeonBaseReminder2', faction = 'UEF'},
+}
+
+
+
+-- When the ship gets damaged
+M1ShipDamaged = {
+    {text = '[Benson]: Commander, the ship is under attack! Protect it!', vid = 'N03_M1ShipDamaged.sfd', bank = 'N03_VO', cue = 'M1ShipDamaged', faction = 'UEF'},
+}
+
+-- When the ship drops to half HP
+M1ShipHalfDead = {
+    {text = '[Benson]: The ship is taking damage!', vid = 'N03_M1ShipHalfDead.sfd', bank = 'N03_VO', cue = 'M1ShipHalfDead', faction = 'UEF'},
+}
+
+-- When the ship is nearly destroyed
+M1ShipAlmostDead = {
+    {text = '[Benson]: The ship is almost dead!', vid = 'N03_M1ShipAlmostDead.sfd', bank = 'N03_VO', cue = 'M1ShipAlmostDead', faction = 'UEF'},
+}
+
+
+
+-- First Aeon units close the the player
+M1AeonAttackWarning = {
+    {text = '[Benson]: Aeon units are closing it to your position, get ready to fight.', vid = 'N03_M1AeonAttackWarning.sfd', bank = 'N03_VO', cue = 'M1AeonAttackWarning', faction = 'UEF'},
+}
+
+
+
+-- Player sees the dead UEF Base
+M1UEFBaseDialogue = {
+    {text = '[Anyone]: Something about the dead base.', vid = 'N03_M1UEFBaseDialogue.sfd', bank = 'N03_VO', cue = 'M1UEFBaseDialogue', faction = 'UEF'},
+}
+
+
+
+-- Once player scouts the civilian city, add objective to capture one building
+M1SecondaryObjective = {
+    {text = '[Benson]: We\'ve located a small civilian settlement south of your position. Capture the marked building to get an additional intel on this area.', vid = 'N03_M1SecondaryObjective.sfd', bank = 'N03_VO', cue = 'M1SecondaryObjective', faction = 'UEF'},
+}
+
+-- When player captured the building
+M1SecondaryDone = {
+    {text = '[Benson]: Building captured...', vid = 'N03_M1SecondaryDone.sfd', bank = 'N03_VO', cue = 'M1SecondaryDone', faction = 'UEF'},
+}
+
+-- When the civ building dies
+M1SecondaryFailed = {
+    {text = '[Benson]: The building is destoryed. No intel for us.', vid = 'N03_M1SecondaryFailed.sfd', bank = 'N03_VO', cue = 'M1SecondaryFailed', faction = 'UEF'},
+}
+
+-- Remind to capture the building
+M1SecondaryReminder = {
+    {text = '[Benson]: Capture the Aeon building to gather extra intel.', vid = 'N03_M1SecondaryReminder.sfd', bank = 'N03_VO', cue = 'M1SecondaryReminder', faction = 'UEF'},
+}
+
+
+
+-- Unlock T2 static shield
+M1ShieldUnlock = {
+    {text = '[Benson]: T2 Shield uploaded.', vid = 'N03_M1ShieldUnlock.sfd', bank = 'N03_VO', cue = 'M1ShieldUnlock', faction = 'UEF'},
```

Comment:
Minor grammar correction: '[Benson]: Commander, I've uploaded the Tech 2 Shield Generator schematic to your ACU.'